bin/update_i18n

```diff
@@ -1,30 +1,39 @@
 #!/usr/bin/env php
 <?php
 
+/*
+ * This file is part of the Behat Gherkin Parser.
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
 $gherkinFile = __DIR__ . '/../vendor/cucumber/cucumber/gherkin/gherkin-languages.json';
-$json  = file_get_contents($gherkinFile);
-$array = array();
+$json = file_get_contents($gherkinFile) ?: '';
+$array = [];
 
-foreach (json_decode($json, true) as $lang => $keywords) {
-    $langMessages = array();
+foreach ((array) json_decode($json, true) as $lang => $keywords) {
+    $langMessages = [];
 
-    foreach ($keywords as $type => $words) {
+    foreach ((array) $keywords as $type => $words) {
```

Comment:
Might be safer to assert that the json decodes to an array / the keywords are an array rather than casting - will give a clearer failure if the upstream file format changes?